src/test/scala/io/citrine/lolo/trees/regression/RegressionTreeTest.scala

```diff
@@ -187,6 +207,6 @@ object RegressionTreeTest {
     new RegressionTreeTest().longerTest()
     new RegressionTreeTest().testCategorical()
     new RegressionTreeTest().testLinearLeaves()
-    new RegressionTreeTest().testShortTreeWithLinearLeaf()
```

Comment:
This is just a driver for human use.  The tests are run whenever they have `@Test` annotations